packages/blockchain-wallet-v4-frontend/src/modals/BuySell/EnterAmount/Checkout/index.tsx

```diff
@@ -161,7 +183,7 @@ const Checkout = (props: Props) => {
       props.buySellActions.setMethod(props.defaultMethod)
     }
 
-    if (!Remote.Success.is(props.data)) {
+    if (!data) {
```

Comment:
I think the correct equivalence is !hasData

Reply:
I'll need to check both, actually. Because the type is not ready for it